chart/values.yaml

```diff
@@ -171,15 +162,13 @@ crawl_retries: 1000
 
 crawler_browser_instances: 2
 
-crawler_requests_cpu: "800m"
-crawler_limits_cpu: "1200m"
+crawler_cpu_per_browser: 600
```

Comment:
Since this and `crawler_memory_per_browser` are handled differently than the other values here, might be good to add comments indicating that the unit used will be `m`/milliCPU?